zebra/main.c

```diff
@@ -365,6 +366,9 @@ int main(int argc, char **argv)
 			break;
 		case 's':
 			rcvbufsize = atoi(optarg);
+			if (rcvbufsize < RCVBUFSIZE_MIN)
+				fprintf(stderr,
+					"rcvbufsize is smaller than recommended value");
```

Comment:
message string: start with a capital letter, end with a newline, include the recommended value to be helpful - why make the user guess?